plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintBasePlugin.kt

```diff
@@ -24,11 +27,27 @@ open class KtlintBasePlugin : Plugin<Project> {
             }
         }
 
+        val kotlinScriptAdditionalPathApplier = { fileCollection: FileCollection ->
```

Comment:
Do you want to be explicit about the type here, using `KotlinScriptAdditionalPathApplier` just for sanities sake? If you don't think it adds any value, feel free to not add it.